run-integration-tests.py

```diff
@@ -26,35 +26,56 @@
 import tempfile
 
 
-# TODO Scala integration tests
-def run_scala_integration_tests(root_dir, package):
-    pass
+def run_scala_integration_tests(root_dir, version):
+    print("##### Running Scala tests on version %s #####" % str(version))
+    clear_artifact_cache()
+    test_dir = path.join(root_dir, "examples", "scala")
+    test_src_dir = path.join(test_dir, "src", "main", "scala", "example")
+    test_classes = [f.replace(".scala", "") for f in os.listdir(test_src_dir)
+                  if f.endswith(".scala") and not f.startswith("_")]
+    with WorkingDirectory(test_dir):
+        for test_class in test_classes:
+            try:
+                cmd = ["build/sbt", "runMain example.%s" % test_class]
+                print("Running Scala tests in %s\n=====================" % test_class)
+                print("Command: %s" % str(cmd))
+                run_cmd(cmd, stream_output=True, env={"DELTA_VERSION": str(version)})
+            except:
+                print("Failed Scala tests in %s" % (test_class))
+                raise
 
 
-def run_python_integration_tests(root_dir, package):
-    print("##### Running Python tests #####")
+def run_python_integration_tests(root_dir, version):
+    print("##### Running Python tests on version %s #####" % str(version))
+    clear_artifact_cache()
     test_dir = path.join(root_dir, path.join("examples", "python"))
     test_files = [path.join(test_dir, f) for f in os.listdir(test_dir)
                   if path.isfile(path.join(test_dir, f)) and
                   f.endswith(".py") and not f.startswith("_")]
     python_root_dir = path.join(root_dir, "python")
     extra_class_path = path.join(python_root_dir, path.join("delta", "testing"))
-    # bintray repo url
+    package = "io.delta:delta-core_2.11:" + version
```

Comment:
should we have a separate `get_scala_version` method to when we eventually move to 2.12